index.html

```diff
@@ -3,16 +3,17 @@
 <html>
     <head>
         <title>The Page</title>
+        <meta charset="UTF-8">
     </head>
     <body> 
     <link rel="stylesheet" href="styles.css">
     <h2>Demo JavaScript in Body</h2>
 
     <p id="demo">A Paragraph</p>
 
-    <button type="button" onclick="myFunction()">Try it</button>
+    <button class="btn" type="button" onclick="myFunction()">Try it</button>
```

Comment:
I would suggest to look for a more appropriate and speaking class name in this particular case